FormalConjectures/ErdosProblems/499.lean

```diff
@@ -0,0 +1,73 @@
+/-
+Copyright (c) 2025 Bhavik Mehta
```

Comment:
Confusingly, this isn't in the README yet (see #100), but the convention for this repo is:
```suggestion
Copyright (c) 2025 The Formal Conjectures Authors.
```
And you can add yourself to the AUTHORS file:)

Reply:
If you merge in the latest `main`, CI will check this for you :)